spec/models/post_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'Validations For the post model' do
+    before(:each) do
+      subject { Post.new(Title: 'The Imposible ', comment_count: 10, like_counter: 34) }
+    end
+
+    before { subject.save }
+
+    it 'Title should be present' do
+      subject.Title = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'Title should not exceed 250 characters' do
+      subject.Title = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'comment_counter should be present' do
+      subject.comment_count = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'like_counter should be present' do
+      subject.like_counter = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'comment_count should allow valid values' do
+      subject.comment_count = 12
+      expect(subject).to_not be_valid
+    end
+  end
```

Comment:
- [ ] Please make sure to test all methods implemented in the `Post` model, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods) requirement.